SeaPublicWebsite/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
       "commandName": "IISExpress",
       "launchBrowser": false,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Local",
+        "ASPNETCORE_ENVIRONMENT": "Development",

```

Comment:
This was stopping the built-in configuration code from loading user secrets - and in ,net core v6 would also mess up developer error pages. The framework recognises `Development` as the local development environment.